mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java

```diff
@@ -84,6 +87,20 @@ public void initObjects() {
 
     }
 
+    public void initExpand() {
+        expandableListView.setOnGroupExpandListener(
+                new ExpandableListView.OnGroupExpandListener() {
+                    int expand_pos = -1;
+                    @Override
+                    public void onGroupExpand(int groupPosition) {
+                        if (expand_pos >= 0 && expand_pos != groupPosition) {
```

Comment:
If the user clicks on the expanded item, it should collapse. Does your code restricts that?

Reply:
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/37215508/71671150-3c68c700-2d98-11ea-8fcb-7468d6c0c178.gif)
 No it does not. It works fine